_includes/contributor/labels.md

```diff
@@ -101,7 +101,7 @@ Labels applied to issues through verification and completion. For details on the
 
 All [contributions to DevDocs](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) receive the following labels:
 
-* `New topic`- New file submissions for content that has never existed on devdocs
-* `Major update` - Significant updates to existing content, such as a new section or example
-* `Technical` - Updates to the code or processes that alter the technical content of the document
-* `Editorial` - Fixes for typos, grammatical inconsistencies, or minor rewrites to correct inaccuracies.
+* `New topic`- New topic submissions for content that has never existed on DevDocs such as tutorials, references, instructions, and so on
```

Comment:
```suggestion
* `New topic`- New topic submissions for content that has never existed in DevDocs, such as tutorials, references, instructions, etc.
```

Reply:
Add "detailed descriptions of functionality" or similar.

Reply:
we changed from using etc... no latin abbreviations

Reply:
Hmm... "and so on" just doesn't seem like nomenclature we'd want to include. 🤷‍♀

Reply:
You're welcome to ask the one who changed it.

Reply:
Personally, I prefer "...and other relevant content."